tests/suites/host_test.function

```diff
@@ -431,6 +431,15 @@ int execute_tests( int argc , const char ** argv )
         return( 1 );
     }
 
+    if( outcome_file_name != NULL )
```

Comment:
It seems to me that this file did not have any reference to `outcome_file_name` in mbedtls-2.16 before thie PR, and the compiler would seem to agree:
```
suites/host_test.function: In function 'execute_tests':

suites/host_test.function:434:9: error: 'outcome_file_name' undeclared (first use in this function)

     if( outcome_file_name != NULL )
```
Should this commit be skipped in the 2.16 backport?

Reply:
Indeed. CI also rejects this commit. I force-pushed a new history without that commit.